src/components/header/index.css

```diff
@@ -558,17 +558,49 @@
   margin: 0;
 }
 
+li {
+  margin-bottom: 1em;
+}
+
+li .tutorial-link {
+  display: inline;
+  padding: 0;
+  font-size: 1em;
+  color: blue;
+}
+
+li .tutorial-link:hover {
+  text-decoration: underline;
+}
+
+.site-link-groups a {
+  display: inline;
+  font-size: 1em;
+  background: white;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  line-height: 2em;
+  cursor: pointer;
+  user-select: none;
+  margin-right: 1em;
+  padding: 0.2em 0.9em;
+}
+
+.site-link-groups a:hover {
+  border-color: #aaa;
+}
+
 /* Media queries */
 @media only screen and (min-width: 1200px) {
   .header-button svg,
   .docs-link svg {
     display: block;
   }
 
-  .header-button,
+  /* .header-button,
```

Comment:
Delete if not needed.

Reply:
@domoritz This was interfering with other `a` tags that's why I removed it. If padding is required I can add it.
Right now, it looks like this
![Screenshot from 2019-03-10 15-18-54](https://user-images.githubusercontent.com/35191225/54073234-df50d180-42aa-11e9-9a1c-03c8a42e21db.png)

Reply:
How was it interfering? I think the padding makes it look better.

Reply:
@domoritz It added `padding: 0 20px` to every `a` tag.

Reply:
I added it again though. It was just interfering with tutorial link which you asked me to remove.